python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -387,6 +391,39 @@ def get_asset_keys_to_resource_props(
     }
 
 
+def get_updated_cli_invocation_params_for_context(
+    context: Union[OpExecutionContext, AssetExecutionContext],
+    manifest: Mapping[str, Any],
+    dagster_dbt_translator: "DagsterDbtTranslator",
+):
```

Comment:
I think we should have an output type here. I'm a fan of named container classes once you get to this number of independent outputs, so it could be like:

```python
@record
class DbtInvocationParams:
    manifest: Manifest
    translator: DagsterDbtTranslator
    selection_args: Sequence[str]
    indirect_selection: bool
```

Reply:
Fair, updated in [4a9ac16](https://github.com/dagster-io/dagster/pull/29650/commits/4a9ac16a113b92be5d856f24c4867e8d97c70f5d)